app/src/main/java/ch/hikemate/app/ui/saved/SavedHikesScreen.kt

```diff
@@ -141,29 +141,31 @@ fun SavedHikesScreen(hikesViewModel: HikesViewModel, navigationActions: Navigati
 @Composable
 private fun PlannedHikes(hikes: List<StateFlow<Hike>>?, hikesViewModel: HikesViewModel) {
   val context = LocalContext.current
-  Text(
-      context.getString(R.string.saved_hikes_screen_planned_section_title),
-      style = MaterialTheme.typography.titleLarge,
-      modifier =
-          Modifier.padding(16.dp).testTag(SavedHikesScreen.TEST_TAG_SAVED_HIKES_PLANNED_TITLE))
-
-  val plannedHikes =
-      hikes?.filter { it.value.plannedDate != null }?.sortedBy { it.value.plannedDate }
-
-  if (plannedHikes.isNullOrEmpty()) {
-    Box(modifier = Modifier.fillMaxSize(), contentAlignment = Alignment.Center) {
-      Text(
-          text = context.getString(R.string.saved_hikes_screen_planned_section_empty_message),
-          style = MaterialTheme.typography.bodyLarge,
-          modifier =
-              Modifier.padding(16.dp)
-                  .testTag(SavedHikesScreen.TEST_TAG_SAVED_HIKES_PLANNED_EMPTY_MESSAGE))
-    }
-  } else {
-    LazyColumn {
-      items(plannedHikes.size, key = { plannedHikes[it].value.id }) { index ->
-        val hike by plannedHikes[index].collectAsState()
-        SavedHikeCardFor(hike, hikesViewModel)
+  Column(modifier = Modifier.fillMaxSize()) {
```

Comment:
I'd suggest adding a comment like this to prevent this bug from happening again with further modifications to this file.
```suggestion
// Use fillMaxSize() to ensure consistent layout when switching between tabs
// This prevents unwanted UI resizing that can occur when sections have different
// numbers of items, particularly when transitioning between saved and planned hikes
  Column(modifier = Modifier.fillMaxSize()) {
```